src/VisualStudio/Core/Def/Implementation/TaskList/ProjectExternalErrorReporter.cs

```diff
@@ -42,6 +42,13 @@ public ProjectExternalErrorReporter(ProjectId projectId, string errorCodePrefix,
 
         private bool CanHandle(string errorId)
         {
+            // make sure we have error id, otherwise, we simple don't support
+            // this error
+            if (errorId == null)
+            {
+                return false;
```

Comment:
Please log an NFW so we can fix the error source generating these corrupt diagnostics.